matchrx.c:50:2: error: implicitly declaring library function 'memset' with type 'void *(void *, int, unsigned long)' [-Werror,-Wimplicit-function-declaration]
--- matchrx.c.orig	2000-06-20 18:12:18.000000000 -0500
+++ matchrx.c	2021-02-14 04:25:15.000000000 -0600
@@ -26,6 +26,7 @@
 #include <regex.h>
 #include <stdarg.h>
 #include <stdlib.h>
+#include <string.h>
 #include "matchrx.h"
 
 int
